master/static/migrations/20240430120203_add-workspace-namespace-bindings-table.tx.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE public.workspace_namespace_bindings (
```

Comment:
I commented this on Amanda's PR too, but I think you both are creating the same table in each of your PRs? would be worth syncing to compromise on an approach and share that code between PRs -- or make a new PR just for the migration

Reply:
Amanda is going to base her branch off of mine, since she needs wksp - ns bindings for her code to work. cc: @amandavialva01

Reply:
Yep! Ws-ns binding CLI commands aren't directly dependent on this, but I'll rebase my branch off of this one and will inevitably remove the duplicated migration code